helm-chart/chartpress.yaml

```diff
@@ -1,23 +1,28 @@
+# For a reference on this configuration, see the chartpress README file.
+# ref: https://github.com/jupyterhub/chartpress
 charts:
   - name: binderhub
     imagePrefix: jupyterhub/k8s-
     repo:
       git: jupyterhub/helm-chart
       published: https://jupyterhub.github.io/helm-chart
-    paths:
-      - ..
```

Comment:
https://github.com/jupyterhub/binderhub/pull/927 is the PR that added this. Check out the issue linked in it for some details on the bug that was fixed by adding this here. What has changed that we don't need this any more?

Reply:
Before, chartpress did not consider changes to the images it depended on so this was a hack to resolve it. But, now chartpress does this in [this code section](https://github.com/jupyterhub/chartpress/blob/ac2fd8a1f9cb7505c0e215c673683f90a11d329d/chartpress.py#L689-L695), so we don't need this workaround.

Reply:
I think that code section is not in chartpress version 0.4.3, that's must be the reason of https://github.com/jupyterhub/binderhub/issues/919#issuecomment-559370276

Reply:
It was meant to use the info about the file-requirements of the images alongside its own, so it should not had mattered. The issue is that this may still be required but wasnt meant to still be required.

There are tests etc setup in chartpress, but ive apparently missed this test case.

Reply:
Sorry, I think I was not clear or I couldn't understand you. 

What I mean is PR https://github.com/jupyterhub/chartpress/pull/68 is not included in v0.4.3. And that PR fixes the problem we have in binderhub right now. I think we need a version upgrade in chartpress.